### ISSUE TITLE
[Broken Website] support.mozilla.org

### ISSUE THREAD

**Issue by alkaris2** [darkreader/darkreader#13579]
### Prerequisites

- [X] I [searched for any existing report](https://github.com/darkreader/darkreader/issues?q=is%3Aissue) about this website issue to avoid opening a duplicate.
- [X] I can reproduce this website issue in a new, unmodified web browser profile with Dark Reader installed as the only extension.

### Website Issue Description

Dark Reader doesn't work at all when on [support.mozilla.org](https://support.mozilla.org), no matter Dark Mode setting you click on to change, it doesn't work at all.

Dynamic, Filter, Filter+ and Static Dark Mode settings do not work when on ANY of [support.mozilla.org](https://support.mozilla.org) pages. I don't know if this is a bug, or if something else that forces Dark Mode to just simply not work at all on these pages.

### Website Address

https://support.mozilla.org

### Steps To Reproduce

Toggle through different Dark Mode filter settings.

### Operating System

Manjaro 24.1.2

### Web Browser name and version

Freifox 131.0.3 (64-bit)

### Dark Reader version

4.9.96

**Comment by Myshor** [comment]
It's restricteded site by Firefox.

To enable access to restricted sites do the steps from Readme:
https://github.com/darkreader/darkreader#enable-dark-reader-on-restricted-pages-on-mozilla-firefox

**Comment by mr-brunes** [comment]
Being able to view support.mozilla.org in dark mode really needs a simple method that :
1 - doesn't expose one to increased security risk
2 - is far shorter/easier to toggle,
3 - matches the current options.

Granted these may be all be Firefox issues rather than DR ones and so outwith the control of this domain.

**Comment by Myshor** [comment]
@mr-brunes 

Normally this site is restricted by Mozilla. Keep in mind if there are some restricted sites in Chrome or other based browsers you won't be able to enable Dark Reader on them no matter what you will do.

So the points you wrote is not for us... If you really want to have everything "easier" you should ask Mozilla Team to make it easier (but keep in mind if they would find our workaround they could make some total block like in Chromium browsers 🤣).

**Comment by mr-brunes** [comment]
@Myshor yes that is as I'd guessed (as just a lay person), but it still seems odd that one can't just 'flip the bits' on any page at a higher abstract level, as one could with a photo negative viewer. But, as I said, I'm just a lay person! ;-)
